src/Atc.Rest.Client/EndpointResponse.cs

```diff
@@ -1,6 +1,9 @@
+using System;
 using System.Collections.Generic;
 using System.Net;
 
+#pragma warning disable SA1402 // File may only contain a single type
```

Comment:
I'm completely fine with disabling that rule globally if you want.

Reply:
Then I will merge this in 😀

Reply:
Let's just keep it as a #pragma